README.md

```diff
@@ -1 +1,17 @@
-# Frends.MySql
\ No newline at end of file
+# Frends.MySql
+Frends Task for connecting to MySql database.
```

Comment:
Task --> Tasks.
The tasks do more than just connect to the database. Description should point that the tasks are for MySQL database related operations.